README.md

```diff
@@ -103,12 +45,19 @@ To update dependencies, run
 
     dep ensure --update && dep prune
 
+## Further reading
+
+To learn more about Cortex, consult the following documents / talks:
+
+- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
+- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
+
 ## <a name="help"></a>Getting Help
 
-If you have any questions regarding Cortex our hosted Prometheus as a service:
+If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
 
-- Invite yourself to the <a href="https://weaveworks.github.io/community-slack/" target="_blank"> #weave-community </a> slack channel.
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> slack channel.
+- Join us on [our Slack channel](https://slack.weave.works) and ask us directly!
 - Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
```

Comment:
Thanks. There were two different URLs in the original version.

I've taken the liberty of capitalizing "Slack".